harness/determined/cli/cli.py

```diff
@@ -280,6 +280,10 @@ def die(message: str, always_print_traceback: bool = False) -> None:
             die(f"Determined Enterprise Edition is required for this functionality: {e}")
         except FeatureFlagDisabled as e:
             die(f"Master does not support this operation: {e}")
+        except CliError as e:
+            if e.e_stack:
+                print(colored(f"Error: {e}", "yellow"), file=sys.stderr)
+            die(f"{e.name}: {e.message}", exit_code=e.exit_code)
```

Comment:
maybe we don't want to always have this prefix.

Reply:
any thoughts on this? I took out the prefix since that'd be the least change in output. we can opt to add them back later

Reply:
Exception class name as a prefix? yeah I suppose we can remove it. if it's the same for all errors there's not much value in it.

Reply:
I had it be separate for argument errors. I guess it could be useful if you wanted to make the cli output a bit more machine readable but it has its own tradeoffs we don't need to worry about it here